src/ir_Gree.cpp

```diff
@@ -564,23 +563,23 @@ stdAc::swingv_t IRGreeAC::toCommonSwingV(const uint8_t pos) {
 
 /// Convert the current internal state into its stdAc::state_t equivilant.
 /// @return The stdAc equivilant of the native settings.
-stdAc::state_t IRGreeAC::toCommon() {
+stdAc::state_t IRGreeAC::toCommon(void) {
   stdAc::state_t result;
   result.protocol = decode_type_t::GREE;
-  result.model = getModel();
-  result.power = getPower();
-  result.mode = toCommonMode(getMode());
-  result.celsius = !getUseFahrenheit();
-  result.degrees = getTemp();
-  result.fanspeed = toCommonFanSpeed(getFan());
-  if (getSwingVerticalAuto())
+  result.model = this->getModel();
+  result.power = this->getPower();
+  result.mode = this->toCommonMode(this->getMode());
+  result.celsius = !this->getUseFahrenheit();
+  result.degrees = this->getTemp();
+  result.fanspeed = this->toCommonFanSpeed(this->getFan());
+  if (this->getSwingVerticalAuto())
```

Comment:
You new(er) version was fine here too. You could even use `_.Model`, `_.Power`, ... etc here too.